v2/anomalydetection/backend-setup.mdx

```diff
@@ -0,0 +1,229 @@
+---
+sidebar_position: 3
+title: Backend Setup
+hide_title: true
+---
+
+# Setting up the Backend
+
+In order to use the anomaly detection service, a request to the following enpoint (based on the region) needs to be made:
+
+**US Region:**
+```typescript
+POST https://security-us-east-1.aws.supertokens.io/v1/security
+```
+
+**EU Region (Ireland):**
+```typescript
+POST https://security-eu-west-1.aws.supertokens.io/v1/security
+```
+
+**APAC Region (Singapore):**
+```typescript
+POST https://security-ap-southeast-1.aws.supertokens.io/v1/security
+```
+
+You can view the HTTP API reference for this endpoint [here](#somelinkhere).
+
+### Retrieving the Request ID
+
+The request ID that has been passed from the frontend is available in the `requestId` property of the input body from each API override.
+
+Below is an example of how to retrieve the request ID from the input body:
+
+```typescript
+const generatePasswordResetTokenPOST = async function (input) {
+    // ... your code here ...
+    const requestId = (await input.options.req.getJSONBody()).requestId;
+    // ... your code here ...
+}
+```
+
+:::important
+The `requestId` should be required when the trying to reset password, sign in or sign up. If the `requestId` is not present, a 400 error should be thrown.
+:::
```

Comment:
this should be done in code, and explained to the user

Reply:
will refer to code snippets

Reply:
done